modules/claims_api/app/sidekiq/claims_api/poa_vbms_updater.rb

```diff
@@ -4,8 +4,9 @@
 
 module ClaimsApi
   class PoaVBMSUpdater < ClaimsApi::ServiceBase
-    def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
+    def perform(power_of_attorney_id, form = 2122) # rubocop:disable Metrics/MethodLength
       poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
+      rep_or_org = form.to_s == '2122A' ? 'representative' : 'serviceOrganization'
```

Comment:
@stiehlrod  I really like this solution.  I wonder if it would be worthwhile just to undo what I update in the `PoaUpdater` with the `extract_poa_code` method and do the same thing to unify these solutions since it is the same scenario?  Obviously just a thought.